spec/graphql/schema/dynamic_members_spec.rb

```diff
@@ -841,6 +854,10 @@ class OtherObject < GraphQL::Schema::Object
         field :f, Int, null: false
       end
 
+      if GraphQL::Schema.use_schema_subset?
+        ThingInterface.orphan_types(OtherObject)
+      end
```

Comment:
This is necessary because, with Warden, it could be found via (unfiltered) `Schema.types`. But Subset doesn't do that, so it needed another way to reference it. 

(Its use in `ThingUnion.possible_types(...)` would make it so that `Schema.add_type_and_traverse` would find it. But Subset doesn't use the result of `add_type_and_traverse`.)